### PR TITLE
2_Pointer_Method added

### PR DESCRIPTION
**Pull request by somya1252**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 

- Add a new line at the EOF
- Make README updates 
- Add a beginner friendly description about your code

File: C-Plus-Plus/other/2_Pointer_Method.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    while (i < j) {
+
+        if (a[i] + a[j] == x){
+            cout<<"The values whose sum is "<<x<<" are "<<a[i]<<" and "<<a[j];
+            return true;
+        }
+
+        // If sum of elements at current
+        // pointers is less, we move towards
+        // higher values by doing i++
```
Comment (HarshCasper) [inline C-Plus-Plus/other/2_Pointer_Method.cpp]: Use Multi-Line Comments

File: C-Plus-Plus/other/2_Pointer_Method.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    cin>>x;
+    if(twoPointerSum(a,n,x)){
+        return 0;
+    }
+    else{
+        cout<<"There is no two values in the array whose sum is "<<x;
+    }
+    cout<<endl;
+    return 0;
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/other/2_Pointer_Method.cpp]: Add a new line at the EOF

Comment (HarshCasper) [inline C-Plus-Plus/other/2_Pointer_Method.cpp]: Add sample I/O

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/other/2_Pointer_Method.cpp (hunk 2 above)
Comment (somya1252) [reply C-Plus-Plus/other/2_Pointer_Method.cpp]: Added

Comment (somya1252) [reply C-Plus-Plus/other/2_Pointer_Method.cpp]: added

**Review by ankitaggarwal23 — COMMENTED** [review]
Code plagiarized from [here](https://www.geeksforgeeks.org/two-pointers-technique/).

**Pull request closed without merging**